api/server/middleware/audit_linux.go

```diff
@@ -150,8 +150,12 @@ func parseRequest(r *http.Request, d *daemon.Daemon) (string, *container.Contain
 	if d != nil {
 		c, err := d.GetContainer(containerID)
 		if err == nil {
+			if c == nil {
+				logrus.Debug("GetContainer returned nil container.")
```

Comment:
Please include the containerID in the debugging message.

Reply:
done